doc/api/debugger.md

```diff
@@ -202,3 +202,23 @@ To start debugging, open the following URL in Chrome:
 ```
 
 [TCP-based protocol]: https://github.com/v8/v8/wiki/Debugging-Protocol
+
+## Trace Event Support for Node.js
+
+Trace Event provides a mechanism to centralize tracing information generated by
+V8, Node core, and userspace code.
+
+Tracing can be enabled by passing the `--enable-tracing` flag when starting a
+Node.js application.
+
+The set of categories for which traces are recorded can be specified using the
+`--enabled-categories` flag followed by a list of comma separated category names.
```

Comment:
Would it make sense to have them all start with --trace-event so its clear they go together.  For example:

--trace-events-enabled
--trace-event-categories

Not sure how this fits with the way we use enabled/disabled for existing options but having all options related to trace-events start with the same prefix might make it easier to know they are related.